conf/messages

```diff
@@ -107,6 +107,9 @@ verifyEmail.lineTwo = You need to click the link in our email within 15 minutes.
 verifyEmail.lineThree.1 = Check your junk folder. If it’s not there we can
 verifyEmail.lineThree.2 = send it again
 
+emailPreference.title = Do you want us to send you emails instead of letters?
+emailPreference.hint = We currently send messages about your VAT by letter.
```

Comment:
It seems like you've added hint text to the message files but then not referenced it anywhere else